pkg/parsers/operatingsystem/operatingsystem_linux.go

```diff
@@ -14,19 +14,28 @@ var (
 
 	// file to check to determine Operating System
 	etcOsRelease = "/etc/os-release"
+
+	// used by stateless systems like Clear Linux
+	altOsRelease = "/usr/lib/os-release"
```

Comment:
what @runcom is trying to say is that you should remove `etcOsRelease` and `altOsRelease` and declare an array with both paths here:

```go
osReleaseFiles = []string{"/etc/os-release", "/usr/lib/os-release"}
```